config/index.js

```diff
@@ -17,6 +17,9 @@ var conf = convict({
     format: [ 'dev', 'test', 'stage', 'prod' ],
     env: 'NODE_ENV'
   },
+  geodb: {
+    dbPath: path.resolve(__dirname, '../node_modules/fxa-geodb/db/cities-db.mmdb')
```

Comment:
`doc: 'Config for geodb location service` ? and I didnt understand the part about `default`